docs/api/curl/UmbrellaConnectorDataExchange.md

```diff
@@ -0,0 +1,223 @@
+```
+
+7. Query Catalog
+```
+curl -L -X POST 'http://dataconsumer-1-controlplane.tx.test/management/v2/catalog/request' \
+-H 'Content-Type: application/json' \
+-H 'X-Api-Key: TEST1' \
+--data-raw '{
+  "@context": {
+    "@vocab": "https://w3id.org/edc/v0.0.1/ns/"
+  },
+  "@type": "CatalogRequest",
+  "counterPartyAddress": "http://dataprovider-controlplane.tx.test/api/v1/dsp",
+  "counterPartyId": "BPNL00000003AYRE",
+  "protocol": "dataspace-protocol-http",
+  "querySpec": {
+    "offset": 0,
+    "limit": 50
+  }
+}'
+```
+We get the {{offer_id}} from the response.
+
+8. Initiate Negotiation
+```
+curl -L -X POST 'http://dataconsumer-1-controlplane.tx.test/management/v2/contractnegotiations' \
+-H 'Content-Type: application/json' \
+-H 'X-Api-Key: TEST1' \
+--data-raw '{
+	"@context": {
+		"@vocab": "https://w3id.org/edc/v0.0.1/ns/"
+	},
+	"@type": "NegotiationInitiateRequestDto",
+	"counterPartyAddress": "http://dataprovider-controlplane.tx.test/api/v1/dsp",
+	"protocol": "dataspace-protocol-http",
+	"policy": {
+		"@context": "http://www.w3.org/ns/odrl.jsonld",
+		"@type": "odrl:Offer",
+		"@id": "{{offer_id}}",
+         "assigner": "BPNL00000003AYRE",
+		"permission": {
+			"odrl:target": "200",
+			"odrl:action": {
+				"odrl:type": "USE"
+			},
+			"odrl:constraint": {
+				"odrl:or": {
+					"odrl:leftOperand": "BusinessPartnerNumber",
+					"odrl:operator": {
+						"@id": "odrl:eq"
+					},
+					"odrl:rightOperand": "BPNL00000003AZQP"
+				}
+			}
+		},
+		"prohibition": [],
+		"obligation": [],
+		"target": "200"
+	}
+}'
+```
+We get {{negotiation_id}} from the response.
+
+9. Get Negotiation By ID
+```
+curl -L -X GET 'http://dataconsumer-1-controlplane.tx.test/management/v2/contractnegotiations/{{negotiation_id}} \
+-H 'X-Api-Key: TEST1'
+```
+You should be able to see in the response, that the _state_ value is equal to _FINALIZED_.
+
+We get {{contractagreement_id}} from the response.
+
+10. Initiate Transfer
+```
+curl -L -X POST 'http://dataconsumer-1-controlplane.tx.test/management/v2/transferprocesses' \
+-H 'Content-Type: application/json' \
+-H 'X-Api-Key: TEST1' \
+--data-raw '{
+  "@context": {
+    "@vocab": "https://w3id.org/edc/v0.0.1/ns/"
+  },
+  "@type": "TransferRequest",
+  "protocol": "dataspace-protocol-http",
+  "counterPartyAddress": "http://dataprovider-controlplane.tx.test/api/v1/dsp",
+  "contractId": "{{contractagreement_id}}",
+  "assetId": "200",
+  "transferType": "HttpData-PULL",
+  "dataDestination":  {
+    "type": "HttpProxy"
+  },
+  "connectorId": "BPNL00000003AZQP",
+  "callbackAddresses": [
+    {
+      "transactional": true,
+      "uri": "{{webhook}}/api/v1/transfers"
+    }
+  ]
+}'
+```
+We get {{transfer_id}} from the response
+
+11. Get Transfer by ID
+```
+curl -L -X GET 'http://dataconsumer-1-controlplane.tx.test/management/v2/transferprocesses/{{transfer_id}}' \
+-H 'X-Api-Key: TEST1'
+```
+You should be able to see in the response that the _state_ value is equal to _STARTED_.
```

Comment:
```suggestion
```

You should be able to see in the response that the _state_ value is equal to _STARTED_.
```